source/codegen/metadata/nifake_non_ivi/functions.py

```diff
@@ -10,6 +10,24 @@
         ],
         'returns': 'int32'
     },
+    'GetCrossDriverSession': {
```

Comment:
All of the tags to make this work are a little verbose. I wanted to keep the codegen logic a little simpler, so I didn't code up mutations to infer that "cross_driver_session" outputs should be tagged as init_methods with a Session grpc_type. Feedback welcome though.

Reply:
Yeah, we're not talking about a lot of functions like this so I think I like the more verbose approach to keep the codegen simpler.

Reply:
One thing to think about is if we can add some metadata validation rules to this - for example, if a parameter has a `cross_driver_session` then the function should be an `init_method` and things like that. At least then we will know when we get the boilerplate wrong...